pydarn/utils/conversions.py

```diff
@@ -83,7 +83,26 @@ def dmap2dict(dmap_records: List[dict]) -> List[dict]:
         dmap_list.append(OrderedDict(dmap_dict))
     return dmap_list
 
+def gate2GroundScatter(y, Re, reflection_height=250):
```

Comment:
You don't want to pass in a constant, instead remove it as a parameter and then import it and use it in the file. Hopefully, that makes sense?